src/workerd/jsg/resource.h

```diff
@@ -793,6 +798,104 @@ struct ResourceTypeBuilder {
   v8::Local<v8::Signature> signature;
 };
 
+void instantiateModule(v8::Isolate*, v8::Local<v8::Context>, v8::Local<v8::Module>& module);
+
+template<typename TypeWrapper, typename Self>
+struct JsBootstrap {
+  JsBootstrap(jsg::Lock& js, v8::Local<v8::Context> context, v8::Local<v8::Object> target)
+  : js(js), context(context) {}
```

Comment:
Nit: since this is capturing the `jsg::Lock&` reference, be sure to use `KJ_DISALLOW_MOVE_OR_COPY` (or whatever the macro is named, forgetting exactly what it is at the moment)